diff --git a/Makefile.install b/Makefile.install
index 6fcf25e..cbc1611 100644
--- a/Makefile.install
+++ b/Makefile.install
@@ -1,7 +1,7 @@
 #
 # Installation paths.
 DESTDIR		?=
-PREFIX		?= /usr/local
+PREFIX		?= /home/yjy/hku/criu/out
 SBINDIR		?= $(PREFIX)/sbin
 MANDIR		?= $(PREFIX)/share/man
 SYSTEMDUNITDIR	?= $(PREFIX)/lib/systemd/system/
@@ -43,6 +43,6 @@ install-criu: criu
 	$(Q) $(MAKE) -C criu install
 .PHONY: install-criu
 
-install: install-man install-lib install-criu
+install: install-lib install-criu
 	@true
 .PHONY: install
